src/classes/EP_ManageEPTemplate_CTRL.cls

```diff
@@ -0,0 +1,629 @@
+        buttonColPanel.childComponents.add(addTaskButton);
+
+        topLevelPanel.childComponents.add(buttonGridPanel);
+
+        return topLevelPanel;
+    }
+
+    /** @description Generates the markup for a single Engagement Plan Task form, then calls itself recursively for that
+    * task's children.
+    * @param wrapper The TaskWrapper for the current Engagement Plan Task.
+    * @param level The number for which child record we're at in the current place of the heirarchy.
+    * @param accessString The string for accessing the current place in the heirarchy using a nested expression.
+    * @param levelString The numbered heirarchy level for naming and accessing EP Tasks.
+    * @return Component.Apex.OutputPanel A form for an EP Task and its children.
+    */
+    private Component.Apex.OutputPanel addChildComponents(TaskWrapper wrapper, Integer level, String accessString, String levelString) {
+        accessString += '[' + level + ']';
+        wrapper.level = levelString;
+        mapTaskWrappers.put(wrapper.level, wrapper);
+
+        //number for looping through the available colors for the side bar
+        Integer colorNumber = Math.mod(wrapper.level.countMatches('-'), sidebarColors.size());
+
+        Component.Apex.OutputPanel topLevelPanel = new Component.Apex.OutputPanel(styleClass='slds-m-left--medium slds-m-top--small', layout='block');
+        Component.Apex.OutputPanel formPanel = new Component.Apex.OutputPanel();
+    
+        //creates a box around the form with a label for the heirarchy level, and a color bar to more easily grock which level we're at
+        Component.Apex.OutputText boxAndLabel = new Component.Apex.OutputText(value='<div aria-labelledby="task' + wrapper.level + 
+            '", class="slds-m-left--medium slds-m-top--small"><fieldset class="slds-box slds-theme--default slds-container--fluid '+
+            'slds-p-around--small", style="border-left: 6px solid ' + sidebarColors[colorNumber] + '"><legend id=task' + wrapper.level + 
+            ' class="slds-text-heading--small">' + Schema.Sobjecttype.Task.getLabel() + ' ' + wrapper.level + '</legend>', escape=false);
+
+        Component.Apex.OutputPanel firstRow = new Component.Apex.OutputPanel(styleClass='slds-grid', layout='block');
+        Component.Apex.OutputPanel secondRow = new Component.Apex.OutputPanel(styleClass='slds-grid', layout='block');
+        Component.Apex.OutputPanel cardFooter = new Component.Apex.OutputPanel(styleClass='slds-grid', layout='block');
+        Component.Apex.OutputPanel buttonGroup = new Component.Apex.OutputPanel(styleClass='slds-button-group slds-p-top--large', layout='block');
+
+        firstRow.childComponents.add(genericInputPanel(wrapper, accessString, 'Name', 'slds-col slds-size--1-of-4', 'slds-input'));
+        firstRow.childComponents.add(genericInputPanel(wrapper, accessString, 'Assigned_To__c', 'slds-col slds-size--1-of-4', 'lookupInput203'));
```

Comment:
you might be able to ignore my last two comments!  in my 2nd DE org, the control appears correctly as a single field.  not sure why my 1st DE org didn't display it correctly, unless the static resource failed to update for some reason.